<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>887</width>
    <height>754</height>
   </rect>
  </property>
  <property name="mouseTracking">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(230, 255, 254);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>21</x>
      <y>41</y>
      <width>151</width>
      <height>37</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);
font: 16pt &quot;SimSun&quot;;
color: rgb(11, 11, 11);
border-radius: 15px;
</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:16pt; font-weight:700; color:#5e1102;&quot;&gt;New idea!&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>391</width>
      <height>261</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(188, 198, 255);
color: rgb(0, 0, 0);
font: 14pt &quot;SimSun&quot;;
border-radius: 10px;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>40</y>
      <width>181</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(11, 11, 11);
font: 14pt &quot;SimSun&quot;;</string>
    </property>
    <property name="text">
     <string>Category:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>40</y>
      <width>191</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(188, 254, 255, 80);
color: rgb(0, 0, 0);
font: 14pt &quot;SimSun&quot;;
border-radius: 5px;</string>
    </property>
    <item>
     <property name="text">
      <string>Non-category</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="newCatBth">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>40</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(188, 254, 255, 80);
color: rgb(0, 0, 0);
font: 14pt &quot;SimSun&quot;;
border-radius: 5px;</string>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QPushButton" name="saveCatBth">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>370</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(188, 254, 255, 80);
color: rgb(0, 0, 0);
font: 14pt &quot;SimSun&quot;;
border-radius: 5px;</string>
    </property>
    <property name="text">
     <string>Save </string>
    </property>
   </widget>
   <widget class="QLineEdit" name="newCatPoint">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>370</y>
      <width>174</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(188, 198, 255, 80);
color: rgb(0, 0, 0);
font: 14pt &quot;SimSun&quot;;
border-radius: 5px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="saveRandBth">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>430</y>
      <width>216</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(188, 254, 255);
color: rgb(0, 0, 0);
font: 14pt &quot;SimSun&quot;;
border-radius: 5px;</string>
    </property>
    <property name="text">
     <string>Save(non-category)</string>
    </property>
   </widget>
   <widget class="QPushButton" name="saveBth">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>430</y>
      <width>111</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(188, 254, 255);
color: rgb(0, 0, 0);
font: 14pt &quot;SimSun&quot;;
border-radius: 5px;</string>
    </property>
    <property name="text">
     <string>Save</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>2</width>
      <height>2</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2"/>
   </widget>
   <widget class="QPushButton" name="myIdeasBth">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>520</y>
      <width>96</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(188, 254, 255, 80);
color: rgb(0, 0, 0);
font: 14pt &quot;SimSun&quot;;
border-radius: 5px;</string>
    </property>
    <property name="text">
     <string>My ideas</string>
    </property>
   </widget>
   <widget class="QPushButton" name="createBth">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>520</y>
      <width>132</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(188, 254, 255, 80);
color: rgb(0, 0, 0);
font: 14pt &quot;SimSun&quot;;
border-radius: 5px;</string>
    </property>
    <property name="text">
     <string>Pinterest</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_2">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>90</y>
      <width>361</width>
      <height>311</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-color: rgb(20, 20, 20);
border-style: solid;
 border-width: 1px;
 border-color: black;
border-radius: 10px;</string>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
